data/sql/updates/pending_db_world/questfix.sql

```diff
@@ -0,0 +1,67 @@
+
+ -- Sir Wendell's Grave smart ai
+SET @ENTRY := 194537;
+DELETE FROM `smart_scripts` WHERE `source_type` = 1 AND `entryOrGuid` = @ENTRY;
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 1, 0, 0, 71, 0, 100, 0, 21077, 0, 0, 0, 12, 33439, 3, 19000, 0, 0, 0, 8, 0, 0, 0, 8461.727, 468.7472, 596.2335, 4.729842, 'On game object event 21077 triggered - Self: Summon creature Sir Wendell Balfour (33439) at (8461.727, 468.7472, 596.2335, 4.729842) as summon type timed despawn with duration 19 seconds');
+
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 194537 AND `SourceId` = 1;
```

Comment:
you can combine these with the other sourceId= 1 with 
```AND `SourceEntry` IN ( 194537, 194539, etc) ```
Same with comibining the sourceid = 0's.

Also, you can keep 1 whiteline between different sets, but it's common to use only 1 whiteline on AC